test/transpile.jl

```diff
@@ -803,6 +931,22 @@ end
     end
 end
 
+@testset "cast_to_cx: Readout" begin
+    transpiler = CastToffoliToCXGateTranspiler()
+
+    circuits = [
+        QuantumCircuit(qubit_count = 2, instructions = [Readout(1)]),
+        QuantumCircuit(qubit_count = 2, instructions = [sigma_x(1), Readout(1)]),
+    ]
+
+    for circuit in circuits
+        transpiled_circuit = transpile(transpiler, circuit)
+
+        @test !circuit_contains_gate_type(transpiled_circuit, Snowflurry.Swap)
+        @test compare_circuits(circuit, transpiled_circuit)
```

Comment:
Same as above

Reply:
updated `compare_circuits` to check `Readouts`